tests/chainer_tests/test_cuda.py

```diff
@@ -194,6 +194,40 @@ def test_cupy_array_async2(self):
         self.assertIsInstance(y, numpy.ndarray)
         cuda.cupy.testing.assert_array_equal(self.x, y)
 
+    def test_single_none(self):
+        assert cuda.to_cpu(None) is None
+
+    def _check_list_tuple(self, typ):
+        assert typ in (list, tuple)
+        a = numpy.random.uniform(-1, 1, (0,))
+        b = numpy.random.uniform(-1, 1, (2, 3))
+        c = cuda.cupy.random.uniform(-1, 1, (0,))
+        d = cuda.cupy.random.uniform(-1, 1, (2, 2))
+        xs = typ([a, b, c, d, None, a, b, None, c, d])
+        xs_cpu = cuda.to_cpu(xs)
+
+        assert isinstance(xs_cpu, typ)
+        assert len(xs) == len(xs_cpu)
+        for i in (0, 1, 2, 3, 5, 6, 8, 9):
+            assert isinstance(xs_cpu[i], numpy.ndarray)
+            cuda.cupy.testing.assert_array_equal(xs[i], xs_cpu[i])
+        assert xs_cpu[0] is a
```

Comment:
Do we need to use `assertIs`?

Reply:
I think `assert` is better, becase it's more straight-forward.
This is one of the benefits we discussed when we moved to PyTest.